### DISCUSSION TITLE
DAS explanation

### DISCUSSION THREAD

**Discussion by tavdog** [emuflight/EmuFlight#493] [Q&A] [answered]
Does das depend on the camera angle value set on config page?  If so, how is it different than normal raw roll mixing?  If not, does it compute based on acc values?

**Comment by nerdCopter** [comment, marked as answer]
https://github.com/emuflight/EmuFlight/wiki/Angle-and-NFE-Racer-modes#mix-roll-and-yaw

The DAS is a firmware-side equivalent of what Whoop pilots have been doing in OpenTX to mix roll and yaw for racing.  The primary benefit is that when it is firmware side, crash handling is superior over TX-side.

Maybe @Quick-Flash would have a better reply for you.

**Reply by tavdog** [reply]
Ok. Thanks for the reply.  Also, how is it different from the cinematic yaw function?

**Reply by Quick-Flash** [reply]
Cinematic yaw applies a roll and yaw compensation based on your pitch angle. Cinematic yaw is similar to FPV Angle Mix, except it has a moving angle based on your pitch angle. DAS is more similar to FPV Angle Mix, than Cinematic yaw. However DAS allows you to configure how much roll inputs will add to yaw or vice versa. FPV Angle Mix is static and forces a certain ratio of roll and yaw stick mixing. This basically allows you to do what mixing roll and yaw in your transmitter does, however this method works a bit better and is less prone to acting weird. I hope that this answer helps.

**Reply by tavdog** [reply]
This answers my question perfectly. Thanks.

**Reply by tavdog** [reply]
Does cinematic yaw only work in angle mode?  I'm trying it in rate mode and it doesn't seem to do anything.

**Reply by Quick-Flash** [reply]
no, its real weird in acro mode, its not worth using in acro.
